### PR TITLE
feat: add example using `rich` and `pixi-build-python` and remove flask

### PR DESCRIPTION
**Pull request by Hofer-Julian**
The other examples where not actually building a Python package with `pixi-build-python`

### CODE REVIEW

**Comment by baszalmstra** [comment]
Wouldnt it make more sense to adapt the `flask-hello-world-pyproject` example as that also contains much more documentation regarding pixi-build?

**Comment by Hofer-Julian** [comment]
> Wouldnt it make more sense to adapt the `flask-hello-world-pyproject` example as that also contains much more documentation regarding pixi-build?

Yeah, probably.

Let me have a look!

File: examples/rich_example/pyproject.toml [hunk 1]
```diff
@@ -1,53 +1,32 @@
...
-
-[tool.pixi.feature.test.tasks]
-test = "pytest -v tests/*"
+[tool.pixi.host-dependencies]
+# To be able to install this pyproject we need to install the dependencies of
+# the python build-system defined above. Note that different from the
+# pyproject build-system this refers to a conda package instead of a pypi
+# package.
+hatchling = "==1.26.3"
+# This way uv is used instead of pip
```
Comment (nichmor) [inline examples/rich_example/pyproject.toml:24]: why do we need uv and how it is used?

**Comment by nichmor** [comment]
looks cleaner and easier to understand. I've also tested `pixi build` and `pixi install` and works as expected

**Review by nichmor — APPROVED** [review]

**Comment by nichmor** [comment]
blocked on windows by missing entry points https://github.com/prefix-dev/rattler-build/blob/cff5ca53751d348a469bfbf9577e970d6797bdc5/docs/tutorials/python.md?plain=1#L101-L105.

**Pull request merged**